File: 2864-maximum-odd-binary-number/2864-maximum-odd-binary-number.py
class Solution(object):
    def maximumOddBinaryNumber(self, s):
        count = 0
        for i in range(0, len(s)):
            if s[i] == '1':
                count += 1;
        
        zero = len(s) - count
        answer = ""
        for i in range(0, count - 1):
            answer = answer + '1'
        
        for i in range(0, zero):
            answer = answer + '0'
        
        return answer + '1'
        

File: Easy/2864-maximum-odd-binary-number/2864-maximum-odd-binary-number.java
class Solution {
    public String maximumOddBinaryNumber(String s) {
        int count = 0;
        for(char ch : s.toCharArray())
        {
            if(ch == '1')
                ++count;
        }
     
        int zero = s.length() - count;
        
        StringBuilder sb = new StringBuilder();
        
        for(int i = 0; i < count - 1; i++)
        {
            sb.append(1);
        }
        
        for(int i = 0; i < zero; i++)
        {
            sb.append(0);
        }
        
        return sb.append(1).toString();
        
    }    
}

// count the number of 1
// 